formats/manual_section/publisher/details.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "body",
+    "manual",
+    "organisations"
+  ],
+  "properties": {
```

Comment:
Yep, I've removed them in the PR: https://github.com/alphagov/specialist-publisher/commit/17ef9957a7bb5cd0e60785bd83e86acd1c23ae7a